scout/adapter/mongo/variant.py

```diff
@@ -560,7 +563,7 @@ def check_causatives(self, case_obj=None, institute_obj=None, limit_genes=None):
             causatives += [
                 var
                 for var in self.match_affected_gt(
-                    other_case, institute_obj, positional_variant_ids, limit_genes
+                    case_obj, institute_obj, positional_variant_ids, limit_genes
```

Comment:
This was an error introduced yesterday by me.

Basically case_obj is the case you are analysing while other_case is the case with the original causative. Of course if you use other case in match_affected_gt it will return the variant, but you are interested in checking of the causative from the other case is found in one affected individual of the case you are analysing... Does it make any sense?